galleries/plot_types/basic/plot.py

```diff
@@ -13,12 +13,16 @@
 
 # make data
 x = np.linspace(0, 10, 100)
-y = 4 + 2 * np.sin(2 * x)
+y = 4 + 1 * np.sin(2 * x)
+x2 = np.linspace(0, 10, 25)
+y2 = 4 + 1 * np.sin(2 * x2)
 
 # plot
 fig, ax = plt.subplots()
 
+ax.plot(x2, y2 + 2.5, 'x', markeredgewidth=2)
```

Comment:
```suggestion
ax.plot(x2, y2 + 2.5, marker='x', markeredgewidth=2)
```
I think nicer to add the keyword name to make it clear what 'x' is doing here? Doing this of course won't turn off the line, and I'm not sure what the canonical way to do that is? So I think work adding `linewidth=0` or another option to explicitly turn off the line drawing here.

Reply:
A key difference between the kwargs and the `fmt` parameter is that the former only modify that aspect (i.e. “add markers”) whereas the latter are absolute (“draw with markers only”). 

The fmt parameter semantics is what we want here, and I believe it’s fundamental enough that we don’t need to jump extra hoops to work around it.